plugins/src/android/OptimoveSDKPlugin.java

```diff
@@ -1,16 +1,205 @@
 package com.optimove.android.cordova;
 
 import org.apache.cordova.CordovaPlugin;
+
+import java.nio.file.WatchEvent;
```

Comment:
Is this import used? Maybe give the file a format & optimise imports in Android Studio.